zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -804,52 +804,67 @@ paths:
           description: >
             The decision requirements search query failed.
             More details are provided in the response body.
-  /authorizations:
-    post:
+
+  /authorizations/{ownerKey}:
+    patch:
       tags:
         - Authorization
-      summary: Create an authorization
-      operationId: createAuthorization
+      summary: Patch an authorization
+      description: Add or remove permissions to an authorization
```

💭 just an option on wording, the add part aligns with `to an authorization` but the `remove permissions` doesn't
```suggestion
      description: Manage the permissions assigned to the authorization
```